docs/www/data-transform.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Data transformation
+order: 0
+---
+# Data transformation
+
+The basic process of streaming data is to have systems produce events to a topic
+and have other systems consume from that topic.
```

Comment:
Opener sounds kinda generic maybe we can improve, @vsaraswat ideas?

Reply:
I agree we should get straight to the point of the feature since most folks know what topics do at this point docs. Suggestion:
Redpanda Data Transforms enable users to create WebAssembly-based scripts to perform simple data transformations on topics. This eliminates the need to consume data streams from a separate system for simple transformations such as scrubbing or cleaning data.